src/braket/device_schema/gate_model_parameters_v1.py

```diff
@@ -40,3 +43,4 @@ class GateModelParameters(BraketSchemaBase):
     )
     braketSchemaHeader: BraketSchemaHeader = Field(default=_PROGRAM_HEADER, const=_PROGRAM_HEADER)
     qubitCount: conint(strict=True, ge=0)
+    explicitQubits: bool = False
```

Comment:
Have you tested this with the SDK and integ tests to SV1?

Reply:
Yep, still work just fine